conda_smithy/lint_recipe.py

```diff
@@ -36,7 +35,12 @@ def lintify(meta):
 
     # 4: The recipe should have some tests.
     if 'test' not in major_sections:
-        lints.append('The recipe must have some tests.')
+        test_files = ['run_test.py', 'run_test.sh', 'run_test.bat', 'run_test.pl']
```

Comment:
I am not sure if there is such a thing. I was actually asking.

@msarahan is there a `run_test` for R?